packages/react-scripts/template/README.md

```diff
@@ -346,6 +346,35 @@ Learn more about ES6 modules:
 * [Exploring ES6: Modules](http://exploringjs.com/es6/ch_modules.html)
 * [Understanding ES6: Modules](https://leanpub.com/understandinges6/read#leanpub-auto-encapsulating-code-with-modules)
 
+## Importing a Component using path relative to src (Absolute path)
+
+Create a file called `.env` in the root of the project, then add the following:
```

Comment:
I recall that Rails' [dotenv](https://github.com/bkeepers/dotenv) (which takes care of loading this `.env` file in a rails app environment) was able to recognize several `.env` files, like `.env`, `.env.local`, `.env.development`, `.env.production`, etc. I think this would fix this issue. The settings to be checked into source control live in `.env`, where a comment can specify that any other settings needed only by a specific developer in their local environment should be set in the `.env.local` file instead.